aider_coder.py

```diff
@@ -22,7 +22,12 @@ def do_coding_request(prompt, files_list, root_folder_path):
     logger.info(f"Files List: {files_list}")
     
     model = Model("claude-3-5-sonnet-20240620")
-    full_file_paths = [os.path.join(root_folder_path, file) for file in files_list]
+    full_file_paths = []
```

Comment:
This isn't quite working. Can we look at the bigger picture of when we are passing in the files list, and add debugging statements to see what is going on?

Reply:
@Aiderbot This isn't quite working. Can we look at the bigger picture of when we are passing in the files list, and add debugging statements to see what is going on?